content/en/docs/Concepts/crds/bundleIndexImage.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Bundle Index Image"
+weight: 3
+---
+
+
+A bundle is an operator packaging construct which contains an operator definition and manifests which ultimately determine how the operator is deployed onto a Kubernetes cluster. Bundles are the preferred mechanism within OLM going forward to package and deploy Operators. Operator developers are encouraged to use the Bundle format over PackageManifests.
```

Comment:
Is the `PackageManifest` format deprecated upstream, or only downstream? If so, we may want to shout out that out, else we can remove that update from the following suggestion:

```suggestion
A bundle is an operator packaging construct that contains an operator definition and the list of manifests needed to determine how the operator will be deployed onto a Kubernetes cluster. Bundles are the preferred mechanism within OLM going forward to package and deploy Operators. Operator developers are encouraged to use the Bundle format over the deprecated PackageManifests format.
```